src/sys/windows/mod.rs

```diff
@@ -37,8 +39,17 @@ impl SerialPort {
 		unsafe {
 			let mut timeouts: winbase::COMMTIMEOUTS = std::mem::zeroed();
 			check_bool(commapi::GetCommTimeouts(file.as_raw_handle(), &mut timeouts))?;
-			timeouts.ReadIntervalTimeout = 10;
-			check_bool(commapi::SetCommTimeouts(file.as_raw_handle(), &mut timeouts))?;
+			// How to set the COMM timeouts so the behavior is a bit more sane
+			// https://learn.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-commtimeouts#remarks
+			// TODO: Expose these as configuration options on Windows
+			timeouts.ReadIntervalTimeout = std::u32::MAX;
+			timeouts.ReadTotalTimeoutMultiplier = std::u32::MAX;
+			// Default timeout is 10ms
+			timeouts.ReadTotalTimeoutConstant = DEFAULT_TIMEOUT;
+			// Set write timeouts
+			timeouts.WriteTotalTimeoutMultiplier = std::u32::MAX;
+			timeouts.WriteTotalTimeoutConstant = DEFAULT_TIMEOUT;
```

Comment:
```suggestion
			timeouts.WriteTotalTimeoutMultiplier = 0;
			timeouts.WriteTotalTimeoutConstant = DEFAULT_TIMEOUT;
```

`DWORDMAX` is not a special value for the write timeouts.

Reply:
This from PySerial win32 seems to show otherwise
```python
        # Set Windows timeout values
        # timeouts is a tuple with the following items:
        # (ReadIntervalTimeout,ReadTotalTimeoutMultiplier,
        #  ReadTotalTimeoutConstant,WriteTotalTimeoutMultiplier,
        #  WriteTotalTimeoutConstant)
        timeouts = win32.COMMTIMEOUTS()
        if self._timeout is None:
            pass  # default of all zeros is OK
        elif self._timeout == 0:
            timeouts.ReadIntervalTimeout = win32.MAXDWORD
        else:
            timeouts.ReadTotalTimeoutConstant = max(int(self._timeout * 1000), 1)
        if self._timeout != 0 and self._inter_byte_timeout is not None:
            timeouts.ReadIntervalTimeout = max(int(self._inter_byte_timeout * 1000), 1)

        if self._write_timeout is None:
            pass
        elif self._write_timeout == 0:
            timeouts.WriteTotalTimeoutConstant = win32.MAXDWORD
        else:
            timeouts.WriteTotalTimeoutConstant = max(int(self._write_timeout * 1000), 1)
        win32.SetCommTimeouts(self._port_handle, ctypes.byref(timeouts))
```

Reply:
```
        elif self._write_timeout == 0:
            timeouts.WriteTotalTimeoutConstant = win32.MAXDWORD
```

Reply:
Set to MAXDWORD to disable timeout entirely

Reply:
Thats an attempt to make the timeout as long as possible. It's not a special value.

Also, I'm suggesting to change the `WriteTotalTimeoutInterval` to 0, not `WriteTotalTimeoutConstant`.